.envsample

```
@@ -0,0 +1,13 @@
+DATABASE_URL="mysql://USERNAME:PASSWORD@HOST:PORT/DATABASE"
```

```suggestion
DATABASE_URL="rdbms://USERNAME:PASSWORD@HOST:PORT/DATABASE"
```
1. file name은 `.env.sample`로 수정해주세요
2. 다른 variables를 중의적으로 잘 표현해주신 것 처럼 `mysql`이라는 것도 `rdbms`로 수정해주세요~